client/data/courses/constants.ts

```diff
@@ -16,4 +17,14 @@ export const COURSE_DETAILS = {
 			i18n.translate( 'Set yourself up for blogging success' ),
 		],
 	},
+	[ COURSE_SLUGS.PAYMENTS_FEATURES ]: {
+		headerTitle: i18n.translate( 'Add Payments Features' ),
```

Comment:
This title seems a bit off to me. I think we also should use a "sentence case" format instead of a "Title Case" format. I noted we have a final stop in the title of the other course, so we should have it as well.

I guess something like "Make money from your website." works better? cc @sethmclellana8c

```suggestion
		headerTitle: i18n.translate( 'Make money from your website.' ),
```